postcss.config.js

```diff
@@ -1,9 +1,5 @@
 /* eslint-disable import/no-commonjs, no-global-assign */
 
-// When the configuration file refers to `window`, we need to shim it so it
-// doesn't break when processed with Node during the build step.
-window = global;
```

Comment:
Any idea why this is no longer needed? Does Node 14 handle it?

Reply:
It could be Node 14, or webpack, or postcss. I looked for an explanation but could not find any concrete information. I got this error message for all scss files, which pointed me to this declaration, and removing it resolved the issue:

```
ERROR in ./src/reset.scss (./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./src/reset.scss)
Module build failed (from ./node_modules/postcss-loader/dist/cjs.js):
ReferenceError: window is not defined
[...]
```